tests/ctst/world/Zenko.ts

```diff
@@ -240,17 +306,23 @@ export default class Zenko extends World {
             // Assume the role and save the credentials
             const ARWWI = await STS.assumeRoleWithWebIdentity(this.options, this.parameters);
             if (ARWWI && typeof ARWWI !== 'string' && ARWWI.stdout) {
-                this.parameters.AssumedSession = JSON.parse(ARWWI.stdout).Credentials;
+                const parsedOutput = JSON.parse(ARWWI.stdout) as { Credentials: ClientOptions['AssumedSession'] };
+                if (parsedOutput && parsedOutput.Credentials) {
+                    this.parameters.AssumedSession = parsedOutput.Credentials;
+                }
             } else {
                 throw new Error('Error when trying to Assume Role With Web Identity.');
             }
             // Save the session for future scenarios (increases performance)
-            CacheHelper.ARWWI[ARWWIName] = this.parameters.AssumedSession;
-            this.cliMode.parameters.AssumedSession = CacheHelper.ARWWI[ARWWIName];
+            CacheHelper.ARWWI[ARWWIName] = this.parameters.AssumedSession as ClientOptions['AssumedSession'];
```

Comment:
Same here, it can be set in the paramaters type above, to avoid casting here

Reply:
b7460b1c75e76aefa5df57099a8cbecb7c46e999